lib/app/view/app_scaffold.dart

```diff
@@ -1,6 +1,9 @@
 import 'package:ax_dapp/app/widgets/widgets.dart';
+import 'package:ax_dapp/wallet/view/view.dart';
 import 'package:flutter/foundation.dart';
+import 'package:flutter/gestures.dart';
 import 'package:flutter/material.dart';
+import 'package:go_router/go_router.dart';
```

Comment:
Are we using this import in this file? If not, I suggest checking all the files that you changed and make sure to remove any other unused imports